### ISSUE TITLE
[Typescript] Allow for getOrders flexibility without throwing an error

### ISSUE THREAD

**Issue by dhstack** [alpacahq/alpaca-trade-api-js#182]
Thinking the types for `getAll()` needs updated to allow for not including ALL the parameters. They should all be optional. I believe the file is `order.d.ts`.

I'm expecting to write code like this:
`openOrders = await alpaca.getOrders({ status: "open", limit: "500" });`

Without getting this error:

```
error TS2345: Argument of type '{ status: string; limit: string; }' is not assignable to parameter of type '{ status: any; until: any; after: any; limit: any; direction: any; nested: any; }'.
  Type '{ status: string; limit: string; }' is missing the following properties from type '{ status: any; until: any; after: any; limit: any; direction: any; nested: any; }': until, after, direction, nested
```

I think all it needs is some question marks after the optional parameters: https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters

**Comment by dhstack** [comment]
Yep, just tried it by adding question marks to all the parameters and it worked!
![image](https://user-images.githubusercontent.com/2013388/148456063-cad0d0ad-6eab-4c83-b2cd-381c46cc01e2.png)

**Comment by Azein** [comment]
The issue should be fixed in the upcoming 4.0 release: https://github.com/alpacahq/alpaca-trade-api-js/blob/ts-alpha/src/trading/apis/OrdersApi.ts#L38, https://github.com/alpacahq/alpaca-trade-api-js/blob/ts-alpha/src/trading/apis/OrdersApi.ts#L238, it is possible to try here: https://www.npmjs.com/package/@alpacahq/alpaca-trade-api/v/4.0.0-alpha.0

Issue remains open until 4.0 is live.

**Comment by Azein** [comment]
4.0 is live, closing the issue.
